legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/finos/FinosGitlabProjectStructureExtensionProvider.java

```diff
@@ -38,9 +38,9 @@ protected FinosGitlabProjectStructureExtensionProvider()
     @Override
     public Integer getLatestVersionForProjectStructureVersion(int projectStructureVersion)
     {
-        if (projectStructureVersion == 11)
+        if (projectStructureVersion >= 11)
```

Comment:
It's better not to use inequalities in this code. It will get us in trouble down the road.